src/WebJobs.Script/Description/FunctionDescriptorProvider.cs

```diff
@@ -314,58 +324,34 @@ protected ParameterDescriptor ParseTimerTrigger(TimerBindingMetadata trigger, Ty
             return new ParameterDescriptor(parameterName, triggerParameterType, attributes);
         }
 
-        protected ParameterDescriptor ParseHttpTrigger(HttpTriggerBindingMetadata trigger, Collection<CustomAttributeBuilder> methodAttributes, Type triggerParameterType = null)
+        protected ParameterDescriptor ParseHttpTrigger(HttpTriggerBindingMetadata trigger, Type triggerParameterType = null)
         {
             if (trigger == null)
             {
                 throw new ArgumentNullException("trigger");
             }
 
-            if (methodAttributes == null)
-            {
-                throw new ArgumentNullException("methodAttributes");
-            }
-
             if (triggerParameterType == null)
             {
                 triggerParameterType = typeof(string);
             }
 
-            ConstructorInfo ctorInfo = typeof(NoAutomaticTriggerAttribute).GetConstructor(new Type[0]);
-            CustomAttributeBuilder attributeBuilder = new CustomAttributeBuilder(ctorInfo, new object[0]);
-            methodAttributes.Add(attributeBuilder);
-
-            ctorInfo = typeof(TraceLevelAttribute).GetConstructor(new Type[] { typeof(TraceLevel) });
```

Comment:
Also removing this TraceLevel annotation we were adding for Http functions to turn off Dashboard logging. With @MikeStall 's recent logging changes, we don't need this anymore.